### PR TITLE
feat: update tests to work on vue boilerplate

### PR DESCRIPTION
**Pull request by MaciejWolowski**
### ❓ Type of change

- [x ] 📖 Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ x] 👌 Enhancement (improving an existing functionality like performance)
- [x ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Updated tests to work with PRs and checking vue boilerplate

### 📝 Checklist

- [ x] I have linked an issue or discussion.
- [x ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/web/__tests__/support/pageObjects/CheckoutPageObject.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { TIMEOUT } from 'dns';
```
Comment (Szymon-dziewonski) [inline apps/web/__tests__/support/pageObjects/CheckoutPageObject.ts:1]: its not used, can be removed probably :)

File: apps/web/__tests__/support/pageObjects/ProductDetailPageObject.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ export class ProductDetailPageObject {
     this.productTitle.should('have.text', data.name);
     this.productDescription.should('have.text', data.description);
     this.productGallery.should('be.visible');
-    this.productPriceValue.should('have.text', data.currency + data.price.toFixed(2));
+    this.productPriceValue.contains(  data.currency + data.price.toFixed(2));
```
Comment (Szymon-dziewonski) [inline apps/web/__tests__/support/pageObjects/ProductDetailPageObject.ts:40]:
```suggestion
    this.productPriceValue.contains(data.currency + data.price.toFixed(2));
```

File: package.json [hunk 3]
```diff
@@ -36,6 +36,7 @@
     "babel-preset-node": "^5.1.1",
     "commitizen": "^4.2.5",
     "cypress": "^12.12.0",
+    "cypress-wait-until": "^2.0.0",
```
[the remark this answers is not in the document]
Comment (MaciejWolowski) [reply package.json:39]: Its Boss idea :P

File: apps/web/__tests__/support/pageObjects/CheckoutPageObject.ts (hunk 1 above)
Comment (MaciejWolowski) [reply apps/web/__tests__/support/pageObjects/CheckoutPageObject.ts]: deleted

**Review by Szymon-dziewonski — APPROVED** [review]

**Pull request merged**